index.html

```diff
@@ -42,7 +49,7 @@ <h1 class="greetings">I’m Abraham, Glad to see you!</h1>
           I’m a software developer! I can help you build a product , feature or
           website Look through some of my work and experience! If you like what
           you see and have a project you need coded, don’t hestiate to contact
-          me. LET’S CONNECT
+          me.
         </p>
         <p class="connect">LET’S CONNECT</p>
         <ul class="connect-links">
```

Comment:
- [ ] Kindly check on your mobile design on this section the images are not arranged as per the design below is a comparison;


Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/171621596-fd35f0bc-0804-4057-a632-585fd5f65f2f.png) |  ![image](https://user-images.githubusercontent.com/61408860/171621646-4aea4a10-dd51-4478-9251-e64577c8d7fd.png)

Reply:
@Ibizugbe, as requested by the last reviewer, the images are not arranged as per the design of the Figma template of your choosing. As you can see, the navbar does not contain any background image, only the section below it does. Please consider this change request.